docs/source/operations/truenas-storage-for-zero-cluster.md

```diff
@@ -0,0 +1,645 @@
+   Mediasize: 899627876352 (838G)
+   Sectorsize: 512
+   Mode: r2w2e5
+   descr: LSI MR9286CV-8e
+   lunid: 600605b009275cc01be6df5e1c7af946
+   ident: 0046f97a1c5edfe61bc05c2709b00506
+   rotationrate: unknown
+   fwsectors: 63
+   fwheads: 255
+
+Geom name: da30
+Providers:
+1. Name: da30
+   Mediasize: 899627876352 (838G)
+   Sectorsize: 512
+   Mode: r2w2e5
+   descr: LSI MR9286CV-8e
+   lunid: 600605b009275cc01be6df611ca0e7c4
+   ident: 00c4e7a01c61dfe61bc05c2709b00506
+   rotationrate: unknown
+   fwsectors: 63
+   fwheads: 255
+
+Geom name: da31
+Providers:
+1. Name: da31
+   Mediasize: 899627876352 (838G)
+   Sectorsize: 512
+   Mode: r2w2e5
+   descr: LSI MR9286CV-8e
+   lunid: 600605b009275cc01be6df631cc6f043
+   ident: 0043f0c61c63dfe61bc05c2709b00506
+   rotationrate: unknown
+   fwsectors: 63
+   fwheads: 255
+
+Geom name: da32
+Providers:
+1. Name: da32
+   Mediasize: 899627876352 (838G)
+   Sectorsize: 512
+   Mode: r2w2e5
+   descr: LSI MR9286CV-8e
+   lunid: 600605b009275cc01be6df661ced8d32
+   ident: 00328ded1c66dfe61bc05c2709b00506
+   rotationrate: unknown
+   fwsectors: 63
+   fwheads: 255
+
+Geom name: da33
+Providers:
+1. Name: da33
+   Mediasize: 899627876352 (838G)
+   Sectorsize: 512
+   Mode: r2w2e5
+   descr: LSI MR9286CV-8e
+   lunid: 600605b009275cc01be6df681d144568
+   ident: 006845141d68dfe61bc05c2709b00506
+   rotationrate: unknown
+   fwsectors: 63
+   fwheads: 255
+
+Geom name: da34
+Providers:
+1. Name: da34
+   Mediasize: 899627876352 (838G)
+   Sectorsize: 512
+   Mode: r2w2e5
+   descr: LSI MR9286CV-8e
+   lunid: 600605b009275cc01be6df6b1d3b9a53
+   ident: 00539a3b1d6bdfe61bc05c2709b00506
+   rotationrate: unknown
+   fwsectors: 63
+   fwheads: 255
+
+Geom name: da35
+Providers:
+1. Name: da35
+   Mediasize: 899627876352 (838G)
+   Sectorsize: 512
+   Mode: r2w2e5
+   descr: LSI MR9286CV-8e
+   lunid: 600605b009275cc01be6df6e1d631e60
+   ident: 00601e631d6edfe61bc05c2709b00506
+   rotationrate: unknown
+   fwsectors: 63
+   fwheads: 255
+
+Geom name: da36
+Providers:
+1. Name: da36
+   Mediasize: 899627876352 (838G)
+   Sectorsize: 512
+   Mode: r2w2e5
+   descr: LSI MR9286CV-8e
+   lunid: 600605b009275cc01be6df711d925523
+   ident: 002355921d71dfe61bc05c2709b00506
+   rotationrate: unknown
+   fwsectors: 63
+   fwheads: 255
+
+Geom name: da37
+Providers:
+1. Name: da37
+   Mediasize: 899627876352 (838G)
+   Sectorsize: 512
+   Mode: r2w2e5
+   descr: LSI MR9286CV-8e
+   lunid: 600605b009275cc01be6df731dbac0b0
+   ident: 00b0c0ba1d73dfe61bc05c2709b00506
+   rotationrate: unknown
+   fwsectors: 63
+   fwheads: 255
+
+Geom name: da38
+Providers:
+1. Name: da38
+   Mediasize: 899627876352 (838G)
+   Sectorsize: 512
+   Mode: r2w2e5
+   descr: LSI MR9286CV-8e
+   lunid: 600605b009275cc01be6df761de3bf6c
+   ident: 006cbfe31d76dfe61bc05c2709b00506
+   rotationrate: unknown
+   fwsectors: 63
+   fwheads: 255
+
+```
+
+## Zpool configuration
+
+The drives are configured as a set of three RAIDZ2 vdevs with two hot
```

Comment:
looks like there are 4 sets not three? unless I am not understanding it correctly.